src/cli/invocation/command/support/embeddedCommandExecution.ts

```diff
@@ -60,7 +60,7 @@ export interface EmbeddedCommandSpec {
  * being exposed as optional command parameters.
  * @param {yargs.Argv} yargs
  */
-export function addEmbeddedCommand(yargs: Argv,
+export function addEmbeddedCommand(yargs: YargSaver,
```

Comment:
I don't particularly like "yargs" used for a `YargSaver`. It's confusing re yargs the project